### DISCUSSION TITLE
VOC finetuning

### DISCUSSION THREAD

**Discussion by johanenforcer** [ultralytics/yolov5#2673] [Q&A]
When using the available VOC dataset for finetuning I noticed that the test set is used as the validation set, is there no need for an "actual" validation set or is the test set located somewhere else?

Thanks in advance!

**Comment by glenn-jocher** [comment]
@johanenforcer the actual datasets merged and used are here. The full download script is
https://github.com/ultralytics/yolov5/blob/master/data/scripts/get_voc.sh

```bash
# Download/unzip images and labels
d='.' # unzip directory
url=https://github.com/ultralytics/yolov5/releases/download/v1.0/
f1=VOCtrainval_06-Nov-2007.zip # 446MB, 5012 images
f2=VOCtest_06-Nov-2007.zip     # 438MB, 4953 images
f3=VOCtrainval_11-May-2012.zip # 1.95GB, 17126 images
for f in $f3 $f2 $f1; do
  echo 'Downloading' $url$f '...' 
  curl -L $url$f -o $f && unzip -q $f -d $d && rm $f & # download, unzip, remove in background
done
wait # finish background tasks
```
